tests/test_photoCalib.py

```diff
@@ -65,11 +73,18 @@ def setUp(self):
         self.pointYShift = lsst.geom.Point2D(0, -10)
         self.bbox = lsst.geom.Box2I(lsst.geom.Point2I(-100, -100), lsst.geom.Point2I(100, 100))
 
-        # calibration and instFlux designed to produce calibrated flux of 1.
+        # calibration and instFlux1 are selected to produce calibrated flux of 1.
         self.calibration = 1e-3
         self.calibrationErr = 1e-4
         self.instFlux = 1000.
         self.instFluxErr = 10.
+        self.flux = 1.0
+        self.mag = nJyToMagnitude(self.flux)
+
+        # useful reference points: 575.44 nJy ~= 24.5 mag, 3630.78 * 10^9 nJy ~= 0 mag
+        self.flux2 = 575.439937337159
+        self.instFlux2 = self.instFlux*self.flux2
```

Comment:
I don't see any reference to `instFlux1` in the code, and if `instFlux1` was a calibrated flux of 1, then `instFlux2` should be a calibrated flux of 2...?

Reply:
You have no idea how long I dithered on this one. I can change `instFlux` to `instFlux1` if you think that's better: I just wanted two different values to test, one being "trivial" and the other not so trivial. Should I call the second one `flux575`?

Reply:
I think "1" and "2" (or even "0" and "1") are fine, but the comment should match the name of the variable.